Service/RequestMiddleware.js

```diff
@@ -0,0 +1,5 @@
+const log = require('../Config/winston');
+ 
+module.exports = function(req, res){ 
+    log.info(`Request ${req.method} ${req.url}, IP: ${req.ip}`);
```

Comment:
we also want to log responses and have an ability to match response with corresponding request